frontend/src/components/ConsultantRows.tsx

```diff
@@ -26,54 +28,54 @@ export default function ConsultantRows({
 
   return (
     <>
-      <tr>
+      <tr
+        className="h-[52px]"
+        onMouseEnter={() => setIsButtonHovered(true)}
+        onMouseLeave={() => setIsButtonHovered(false)}
+      >
+        <td
+          className={`rounded-lg border-l-2 w-[10%] ${
+            isListElementVisible
+              ? "border-l-secondary_default"
+              : isButtonHovered
+              ? "border-l-primary_default"
+              : "border-l-primary_l4"
+          } `}
+        >
+          <button
+            className={`p-2 rounded-lg ml-2 hover:bg-primary_default hover:bg-opacity-10 ${
+              isListElementVisible && "rotate-180"
+            }`}
+            onClick={toggleListElementVisibility}
+          >
+            <ChevronDown className={`text-primary_default w-6 h-6`} />
+          </button>
+        </td>
         <td>
-          <div className="flex flex-row gap-2 ">
-            <div
-              className={`py-0 w-0.5 rounded-lg ${
-                isButtonHovered ? "bg-primary_default" : "bg-primary_l4"
-              } ${isListElementVisible && "bg-secondary_default"}`}
-            ></div>
-            <div
-              className="flex flex-row gap-2 items-center h-[52px]"
-              onMouseEnter={() => setIsButtonHovered(true)}
-              onMouseLeave={() => setIsButtonHovered(false)}
+          <div className="flex flex-col gap-1 ">
+            <p
+              className={`text-black text-start ${
+                isListElementVisible ? "body-bold" : "body"
+              }`}
             >
-              <button
-                className={`p-2 rounded-lg hover:bg-primary_default hover:bg-opacity-10 ${
-                  isListElementVisible && "rotate-180"
-                }`}
-                onClick={toggleListElementVisibility}
-              >
-                <ChevronDown className={`text-primary_default w-6 h-6`} />
-              </button>
-              <div className="flex flex-col gap-1">
-                <p
-                  className={`text-black text-start ${
-                    isListElementVisible ? "body-bold" : "body"
-                  }`}
-                >
-                  {consultant.name}
-                </p>
-                <p className="detail text-neutral_l1 text-start">
-                  {`${consultant.yearsOfExperience} års erfaring`}
-                </p>
-              </div>
-            </div>
+              {consultant.name}
+            </p>
+            <p className="detail text-neutral_l1 text-start">
+              {`${consultant.yearsOfExperience} års erfaring`}
+            </p>
           </div>
         </td>
         {consultant.bookings?.map((b) => (
```

Comment:
Spørsmål for læring: på consultant her (og lignende eks: consultants som vi mapper på filtreringen), bør vi egentlig ha noen sjekker på at disse ikke er null eller undefined, eller blir dette sikret av andre deler av koden? 😅

Reply:
Kan være jeg sier noe feil nå, men slik backend API kallet er satt opp, tillater ikke den å sende null/undefined verdier til frontend på api kallet for consultants. Så med mindre noe endrer seg der, skal vi ikke trenge skjekker i koden, siden API-kallet vil returnere 500 hvis det skjer.